packages/loot-core/src/client/queries.ts

```diff
@@ -115,73 +115,61 @@ export function makeTransactionSearchQuery(
   });
 }
 
-export function accountBalance(
-  acct: AccountEntity,
-): Binding<'account', 'balance'> {
+export function accountBalance(acct: AccountEntity) {
   return {
     name: accountParametrizedField('balance')(acct.id),
     query: q('transactions')
       .filter({ account: acct.id })
       .options({ splits: 'none' })
       .calculate({ $sum: '$amount' }),
-  };
+  } satisfies Binding<'account', 'balance'>;
```

Comment:
Curious, is there a particular reason to use `satisfies` instead of annotating a return type?

Reply:
Two reasons: 1) explicit return types can lie (kind of a design flaw in TS..); 2) using `satisfies` allows to narrow down the return type more precisely

Sample https://www.typescriptlang.org/play/?#code/C4TwDgpgBAsiCiA7ArgWygXigbysCAzsAFxQBGA9hQDYQCGiUAvlAD5REBOAlogOYBuAFBCAZskQBjYNwqNUIACqFgAeUQQAFAEpScJGhxCoJqJwjBknRrnxFSwTsmhNhTEeKky5UBcqKKAO4UOkamZhZWNngqDk4uHHQyBKLchLAIKKhuIgD0uVCKABbQ5nTUMUQExEJ+Kupa2gB0dsBQ+VAAPACEALS9eEXcBFAAJhTpiBRtwZwA1rVKKkEhza3tBT39g8NQs3MjAFbIRFCpGkA